src/tests/tests/first_parties.js

```diff
@@ -0,0 +1,32 @@
+(function() {
+  QUnit.test("test twitter content script", (assert) => {
+    const NUM_CHECKS = 1;
+    let done = assert.async(NUM_CHECKS);
+
+    // create element as we'd get it from twitter
+    let tco = "http://t.co/beach-detour/";
+    let destination = "https://the.beach/";
+    let el = document.createElement("div");
+    el.setAttribute("href", tco);
+    el.setAttribute("data-expanded-url", destination);
+    document.querySelectorAll = function () {
```

Comment:
We never restore the original, so it stays overwritten for all subsequent tests. Also, might as well use `sinon.stub` to overwrite and restore, since we already use Sinon.